qa/pages/GenericTestPage.ts

```diff
@@ -57,12 +58,12 @@ export default class GenericTestPage {
   }
 
   async assertOnTestPage() {
-    await expect(this.page).toHaveURL('/chris-testing-page')
+    await expect(this.page).toHaveURL(`${BASE_URL}chris-testing-page`)
```

Comment:
Playwright suggests using regex for this

https://playwright.dev/docs/api/class-pageassertions